src/axom/primal/tests/primal_boundingbox.cpp

```diff
@@ -491,6 +503,13 @@ TEST(primal_boundingBox, bb_scale)
   QBBox bbox4(bbox);
   bbox4.scale(-1.);
   EXPECT_EQ(bbox, bbox4);
+
+  // Show that scaling an invalid bounding box has no effect.
+  QBBox bbox5, bbox6;
+  EXPECT_FALSE(bbox5.isValid());
+  EXPECT_FALSE(bbox6.isValid());
+  bbox5.scale(1.5);
+  EXPECT_EQ(bbox5, bbox6);
```

Comment:
Is this a new thing, that invalid boxes compare equal?  Just asking because it could make sense that they shouldn't compare equal.  Maybe we have a precedence for this already, but I didn't see this behavior mentioned in the doxygen comments.